script.js

```diff
@@ -0,0 +1,52 @@
+
+//	Ensure javascript is loaded after html/css
+$(document).ready(function() {
+	$('form').submit(function(event){
+		event.preventDefault();
+
+//	Establish variable for storage of entries
+
+		var city = $("#city-type").val();
+		var city = city.toLowerCase();
+
+//	Convert all entered strings to Lower Case
+
+
+
+//	Add Classes when certain search terms are invoked
+
+		if (city === 'la'|| city === "los Angeles" || city === "la") {
```

Comment:
You have `city === 'la'` twice. You can just write this line as:
```js
if (city === 'la'|| city === 'los Angeles') {
```
Also, try to be consistent with quotes. Use either `'` or `"`. Try not to mix and match.